src/containers/layout/footer/Footer.styles.ts

```diff
@@ -6,14 +6,20 @@ export const styles = {
   },
   container: {
     display: 'flex',
+    position: 'relative',
     flexDirection: { xs: 'column-reverse', sm: 'row' },
     justifyContent: { xs: 'space-between', sm: 'space-around' },
     alignItems: 'center',
     py: { xs: '12px', sm: '26px' }
   },
+  title: {
+    position: { xs: 'static', sm: 'absolute' },
+    marginLeft: { xs: '0', sm: '-60vw' }
```

Comment:
why margin -60vw? do container max-width as in Figma , position title absolute and left 0